netlify.toml

```diff
@@ -8,18 +8,35 @@
   YARN_FLAGS = "--ignore-scripts"
 
 # https://docs.netlify.com/configure-builds/file-based-configuration/#redirects
+
+[[redirects]]
+  from = "/docs"
+  to = "/getting-started"
+
+[[redirects]]
+  from = "/docs/development/architecture/asts"
+  to = "/blog/asts-and-typescript-eslint"
```

Comment:
FYI, Netlify can use a separate `_redirects` file (https://docs.netlify.com/routing/redirects/), which IMO is more compact and maintainable

Reply:
Oh, that is nicer! Thanks 😄

Reply:
Oh but the `.toml` version has more options available? Now I'm conflicted...